tests/example_experiments_test.py

```diff
@@ -0,0 +1,95 @@
+        question_answer_lstm = self.multiple_choice_qa_dir + "question_answer_lstm.json"
+        self.check_experiment_type_can_be_parsed(question_answer_lstm)
+
+    def test_tuple_entailment_can_be_parsed(self):
+        self.write_tuple_inference_files()
+        svo_mc_tuple_entailment = self.multiple_choice_qa_dir + "svo_mc_tuple_entailment.json"
+        self.check_experiment_type_can_be_parsed(svo_mc_tuple_entailment)
+
+    def test_tuple_inference_can_be_parsed(self):
+        self.write_tuple_inference_files()
+        tuple_inference = self.multiple_choice_qa_dir + "tuple_inference.json"
+        self.check_experiment_type_can_be_parsed(tuple_inference)
+
+    def test_bidaf_can_be_parsed(self):
+        self.write_span_prediction_files()
+        bidaf_squad = self.reading_comprehension_dir + "bidaf_squad.json"
+        self.check_experiment_type_can_be_parsed(bidaf_squad)
+
+    def test_ga_reader_can_be_parsed(self):
+        self.write_who_did_what_files()
+        gareader_who_did_what = self.reading_comprehension_dir + "gareader_who_did_what.json"
+        self.check_experiment_type_can_be_parsed(gareader_who_did_what)
+
+    def test_as_reader_can_be_parsed(self):
+        self.write_who_did_what_files()
+        as_reader_who_did_what = self.reading_comprehension_dir + "asreader_who_did_what.json"
+        self.check_experiment_type_can_be_parsed(as_reader_who_did_what)
+
+    def test_siamese_sentence_selection_can_be_parsed(self):
+        self.write_sentence_selection_files()
+        siamese_sentence_selector_squad = self.sentence_selection_dir + "siamese_sentence_selector_squad.json"
+        self.check_experiment_type_can_be_parsed(siamese_sentence_selector_squad)
+
+    def test_simple_tagger_can_be_parsed(self):
+        self.write_sequence_tagging_files()
+        simple_tagger = self.sequence_tagging_dir + "simple_tagger.json"
+        self.check_experiment_type_can_be_parsed(simple_tagger)
+
+    def check_experiment_type_can_be_parsed(self, param_file):
+        param_dict = pyhocon.ConfigFactory.parse_file(param_file)
+        params = Params(replace_none(param_dict))
+        model_class = concrete_models[params.pop("model_class")]
+        # Tests will try to create root directories as we have /net/efs paths,
+        # so we just remove the serialisation aspect here, alter the train/validation
+        # paths to the dummy test ones and make sure we only do one epoch to
+        # speed things up.
+        params["model_serialization_prefix"] = None
+        if len(params["train_files"]) > 1:
+            params["train_files"] = [self.TRAIN_FILE, self.TRAIN_BACKGROUND]
+            params["validation_files"] = [self.VALIDATION_FILE, self.VALIDATION_BACKGROUND]
+        else:
+            params["train_files"] = [self.TRAIN_FILE]
+            params["validation_files"] = [self.TRAIN_FILE]
+        params["num_epochs"] = 1
+        try:
+            params["embeddings"]["words"]["project"] = False
```

Comment:
How does this work for experiments that used pre-trained embeddings?  It seems like this should be crashing for some of the experiments.  The simple tagger, e.g., should be trying to read the pre-trained embedding file, and it should be crashing because it's not present.  What's going on?